image/templates/helm/stackrox-secured-cluster/templates/collector.yaml.htpl

```diff
@@ -136,6 +136,10 @@ spec:
           valueFrom:
             resourceFieldRef:
               resource: limits.cpu
+        - name: POD_NAME
```

Comment:
Should we also add POD_NAMESPACE for consistency?

Reply:
Good point. I'll add it 👍